apps/teams-test-app/src/components/CallAPIs.tsx

```diff
@@ -15,16 +15,14 @@ const StartCall = (): React.ReactElement =>
     name: 'startCall',
     title: 'Start Call',
     onClick: {
-      validateInput: () => {
-        // TODO: uncomment below once the test is updated to reflect the proper input type
-        /*
+      validateInput: input => {
```

Comment:
Sorry if I missed this on the other PRs- I took a closer look at the argument for this one because of the change you made for line 24. Do we currently not plan to check for optional parameters' validity?

Reply:
I didn't plan to check the optional arguments. Are you suggesting we check that **if** they are provided, they match the types we expect them to be? I'm not sure I find that much value in that. After all it is just a test app. I would definitely do that if we were to write something customers would see, but for an internal tool used mostly for validation it seems like we should only do that if at some point we run into issues that would have been prevented by having that extra argument validation.

Reply:
I agree we don't really need it for our pursposes. I just wanted to make sure we talked about it :) thanks